app/controllers/admin/categories_controller.rb

```diff
@@ -1,13 +1,16 @@
 class Admin::CategoriesController < Admin::BaseController
   cache_sweeper :blog_sweeper
 
-  def index; redirect_to :action => 'new' ; end
+  def index
+     redirect_to :action => 'new' ;
```

Comment:
If you refactor this code to be more than one line, then you should no longer use semi-colons